### ISSUE TITLE
Segfault at Utilities/BEType.h:527

### ISSUE THREAD

**Issue by jclc** [RPCS3/rpcs3#4508]
Emulation regularly ends in this crash. Recently updated Nvidia drivers (390.48-9 -> 390.48-11). Tested on Persona 5 using Vulkan.

Commit: 98b715d8c89f3c57b5e0e9f0a541a53d13e96617
OS: Arch Linux 4.16.5-1
Graphics: GTX 970 390.48-11
CPU: Intel i7-6700K

Backtrace:
```
Core was generated by `/home/vhns/programs/rpcs3-build/bin/rpcs3'.
Program terminated with signal SIGSEGV, Segmentation fault.
#0  se_t<unsigned int, true, 4ul>::operator= (value=<error reading variable: Cannot access memory at address 0x7f5f27effc00>, this=0x1cb17e000) at /home/vhns/programs/rpcs3-git/rpcs3/../Utilities/BEType.h:527
527			return m_data = storage::to(value), *this;
[Current thread is 1 (Thread 0x7f5fa1915700 (LWP 23777))]
(gdb) backtrace
#0  0x0000000000b6f8b7 in se_t<unsigned int, true, 4ul>::operator=(unsigned int) (value=Python Exception <class 'gdb.MemoryError'> Cannot access memory at address 0x7f5f27effc00: 
#1  0x0000000000b6f8b7 in vk::cached_texture_section::do_memory_transfer<unsigned int, true>(void*, void const*, unsigned int) (this=0x7f5f20b24bc8, this=0x7f5f20b24bc8, channels_count=<optimized out>, pixels_src=<optimized out>, pixels_dst=0x1cadfa400) at /home/vhns/programs/rpcs3-git/rpcs3/Emu/RSX/VK/VKTextureCache.h:222
#2  0x0000000000b6f8b7 in vk::cached_texture_section::flush(vk::command_buffer&, VkQueue_T*) (this=0x7f5f20b24bc8, cmd=..., submit_queue=<optimized out>) at /home/vhns/programs/rpcs3-git/rpcs3/Emu/RSX/VK/VKTextureCache.h:293
#3  0x0000000000b6792a in rsx::texture_cache<vk::command_buffer, vk::cached_texture_section, vk::image*, vk::image_view*, vk::image, VkFormat>::flush_all<vk::command_buffer&, VkQueue_T* const&>(rsx::texture_cache<vk::command_buffer, vk::cached_texture_section, vk::image*, vk::image_view*, vk::image, VkFormat>::thrashed_set&, vk::command_buffer&, VkQueue_T* const&) (data=..., this=0x5c1ded0) at /home/vhns/programs/rpcs3-git/rpcs3/Emu/RSX/VK/../Common/texture_cache.h:1192
#4  0x0000000000b6792a in VKGSRender::on_access_violation(unsigned int, bool) (this=0x5c1c660, address=<optimized out>, is_writing=<optimized out>) at /home/vhns/programs/rpcs3-git/rpcs3/Emu/RSX/VK/VKGSRender.cpp:852
#5  0x0000000000d6d6b8 in std::function<bool (unsigned int, bool)>::operator()(unsigned int, bool) const (__args#1=<optimized out>, __args#0=<optimized out>, this=<optimized out>) at /usr/include/c++/7.3.1/bits/std_function.h:706
#6  0x0000000000d6d6b8 in handle_access_violation(unsigned int, bool, ucontext_t*) (addr=addr@entry=3403657216, is_writing=is_writing@entry=true, context=context@entry=0x7f5fa1914180)
    at /home/vhns/programs/rpcs3-git/Utilities/Thread.cpp:1091
#7  0x0000000000d6e9c8 in signal_handler(int, siginfo_t*, void*) (sig=<optimized out>, info=0x7f5fa19142b0, uct=0x7f5fa1914180) at /home/vhns/programs/rpcs3-git/Utilities/Thread.cpp:1510
#8  0x00007f6010676b90 in <signal handler called> () at /usr/lib/libpthread.so.0
#9  0x000000004391ed88 in  ()
#10 0x00007f5fe07da87c in  ()
#11 0x0000000000000000 in  ()
```
[RPCS3.log](https://github.com/RPCS3/rpcs3/files/1965680/RPCS3.log)

Seems to always happen when engaging a certain enemy in the first palace.

**Comment by jclc** [comment]
A driver update seems to have fixed this for now, will reopen the issue if it occurs again

**Comment by jclc** [comment]
Seems to have been fixed as of 0.0.5-6847 and the latest Nvidia drivers.
